common/version/version.go

```diff
@@ -0,0 +1,75 @@
+package version
```

Comment:
We are already vendoring https://github.com/blang/semver. There's also https://github.com/Masterminds/semver, https://github.com/coreos/go-semver and https://github.com/hashicorp/go-version. Do none of these do what we need here?